cli/build.rs

```diff
@@ -0,0 +1,46 @@
+use std::process::{Command, ExitStatus};
+
+fn main() {
+    option_env!("BOBGUI_GIT_HASH").is_none().then(set_hash);
+
+    option_env!("BOBGUI_BUILD_BRANCH_TAG")
+        .is_none()
+        .then(set_branch_tag);
+
+    println!("cargo:rerun-if-changed=../backend");
+}
+
+fn set_hash() {
+    set_env("BOBGUI_GIT_HASH", "git", &["rev-parse", "HEAD"]);
```

Comment:
Declare const with env variabl name. This string used in 2 places in this file. The same for other names